Buildings/Controls/OBC/FDE/DOAS/DehumMode.mo

```diff
@@ -0,0 +1,144 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block DehumMode
+  "This block calculates when dehumidification mode is active."
+
+  parameter Real dehumSet(
+    min=0,
+    max=100)=60
+   "Dehumidification set point.";
+  parameter Real dehumDelay(
+    final unit="s",
+    final quantity="Time")=600
+    "Minimum delay after RH falls below set point before turning dehum off.";
+  parameter Real minRun(
+    final unit="s",
+    final quantity="Time")=120
+    "Minimum supply fan proof delay before allowing dehum mode.";
+
+    // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput retHum
+    "Return humidity sensor."
+      annotation (Placement(transformation(extent={{-142,-20},{-102,20}}),
+        iconTransformation(extent={{-142,-20},{-102,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput supFanProof
+    "True when supply fan is proven on"
+      annotation (Placement(transformation(extent={{-142,18},{-102,58}}),
+        iconTransformation(extent={{-142,-94},{-102,-54}})));
+
+    // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput dehumMode
+    "True when dehumidification mode is active."
+      annotation (Placement(transformation(extent={{102,-12},{142,28}}),
+        iconTransformation(extent={{102,-20},{142,20}})));
+
+  Buildings.Controls.OBC.CDL.Logical.Latch lat
+    annotation (Placement(transformation(extent={{22,-10},{42,10}})));
+  Buildings.Controls.OBC.CDL.Continuous.Greater gre
```

Comment:
add comment, `Check if return air relative humidity is greater than the dehumidificationsetpoint`